pkgs/development/python-modules/bsddb3/default.nix

```diff
@@ -36,6 +34,17 @@ buildPythonPackage rec {
     export BERKELEYDB_DIR=${pkgs.db.dev};
   '';
 
+  postPatch = ''
+    substituteInPlace test3.py \
+      --replace-fail "from distutils.util import get_platform" "import platform" \
+      --replace-fail "get_platform()" "f'{platform.system().lower()}-{platform.machine()}'" \
+      --replace-fail "_TextTestResult" "TextTestResult"
+  '';
```

Comment:
Since we're no longer allowing python >=3.10 this no longer seems necessary.

Reply:
Shouldn't we leave this in case a new package is added in Nixpkgs that do depend on `bsddb3` but does not support `berkeleydb` ? I don't see a reason, why it should be removed, right now, it doesn't break anything to leave it.

Reply:
If the plan is supporting `>=3.10` then keep it, sure.